modules/standard/ChapelIO.chpl

```diff
@@ -657,10 +657,12 @@ module ChapelIO {
       alignCheckRange.normalizeAlignment();
     }
 
-    if hasLowBound() then
+    if (boundedType == BoundedRangeType.bounded ||
+        boundedType == BoundedRangeType.boundedLow) then
```

Comment:
This change seems unnecessary. Please double-check. Ditto hasHighBound() below.

Reply:
It is unnecessary at present, but makes the code more future-proof in the event that we change queries like `hasLowBound()` to return `true` for `..true` or `..violet` in the future.  And I don't think there's any significant downside...?

Reply:
This is OK. The downside is extra change and extra code. In this case it is surely not significant.